docs/releases/2.0.txt

```diff
@@ -305,6 +301,11 @@ Miscellaneous
 * :class:`~django.views.generic.base.RedirectView` no longer silences
   ``NoReverseMatch`` if the ``pattern_name`` doesn't exist.
 
+* :class:`~django.forms.FloatField` and :class:`~django.forms.DecimalField`
+  now respect :setting:`DECIMAL_SEPARATOR` and :setting:`THOUSAND_SEPARATOR`
+  during validation correctly when :setting:`USE_L10N` is off.
```

Comment:
I'm happy for suggestions here @claudep. I don't really see the backwards incompatiblity, the only thing I see is that certain inputs which have been wrongfully rejected as invalid are now accepted.